Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -2089,15 +2102,59 @@ NiftiImageIO::SetImageIOOrientationFromNIfTI(unsigned short dims)
           }();
           prefer_sform_over_qform = sform_and_qform_are_very_similar;
         }
-      } // sform is orthonormal
-    }   // sform not NIFTI_XFORM_UNKNOWN
+      }
+      else if (m_SFORM_Permissive) // sform is orthonormal
```

Comment:
Very minor comment. IMHO, the comment `// sform is orthonormal` marks the end of the previous block. At this line it might be a little bit misleading, IMHO.